uint64 time   # /**< The time stamp of the data processed in the current frame in ms. */
float32 cycleTime  # /**< Length of one cycle in seconds. */
